eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/FooData.Serialization.cs

```diff
@@ -217,7 +217,7 @@ protected virtual ResourceData PersistableModelCreateCore(BinaryData data, Model
         string IPersistableModel<FooData>.GetFormatFromOptions(ModelReaderWriterOptions options) => "J";
 
         /// <param name="fooData"> The <see cref="FooData"/> to serialize into <see cref="RequestContent"/>. </param>
-        public static implicit operator RequestContent(FooData fooData)
+        internal static RequestContent ToRequestContent(FooData fooData)
```

Comment:
if this could be an instance method, it makes more sense since this is seralization - but static method is fine as well.